src/lib/installation/update_repository.rb

```diff
@@ -327,14 +338,15 @@ def build_squashfs(dir, file)
     # @raise CouldNotProbeRepo
     # @raise CouldNotRefreshRepo
     def add_repo
+      return @repo_id unless @repo_id.nil?
```

Comment:
Please update the documentation comment, now it does not add the repo it it has been already added.